src/triggers/issueV2.ts

```diff
@@ -0,0 +1,329 @@
+              id: true,
+              name: true,
+              email: true,
+            },
+            assignee: {
+              id: true,
+              name: true,
+              email: true,
+            },
+            state: {
+              id: true,
+              name: true,
+              type: true,
+            },
+            parent: {
+              id: true,
+              identifier: true,
+              url: true,
+              title: true,
+            },
+          },
+        },
+      },
+    },
+  };
+  const query = jsonToGraphQLQuery(jsonQuery);
+  const response = await fetchFromLinear(z, bundle, query, variables);
+  const data = (response.json as TeamIssuesResponse).data;
+  return data.team.issues.nodes;
+};
+
+const getWebhookDataForIssue = (z: ZObject, bundle: Bundle) => {
+  const entity = {
+    ...bundle.cleanedRequest.data,
+    querystring: undefined,
+  };
+
+  // Webhooks send this over as `milestone`, but the API uses `projectMilestone` and users model their Zaps off the API response to start
+  if (entity.milestone) {
+    entity.projectMilestone = entity.milestone;
+    delete entity.milestone;
+  }
+
+  return [entity];
+};
+
+const operationBase = {
+  inputFields: [
+    {
+      required: true,
+      label: "Team",
+      // This key is snake-cased for backwards compatibility with the dynamic field selector for status, project and label
+      key: "team_id",
+      helpText: "The team for the issue.",
+      dynamic: "team.id.name",
+      altersDynamicFields: true,
+    },
+    {
+      required: false,
+      label: "Status",
+      key: "statusId",
+      helpText: "The issue status.",
+      dynamic: "status.id.name",
+      altersDynamicFields: true,
+    },
+    {
+      required: false,
+      label: "Creator",
+      key: "creatorId",
+      helpText: "The user who created this issue.",
+      dynamic: "user.id.name",
+      altersDynamicFields: true,
+    },
+    {
+      required: false,
+      label: "Assignee",
+      key: "assigneeId",
+      helpText: "The assignee of this issue.",
+      dynamic: "user.id.name",
+      altersDynamicFields: true,
+    },
+    {
+      required: false,
+      label: "Priority",
+      key: "priority",
+      helpText: "The priority of the issue.",
+      choices: [
+        { value: "0", sample: "0", label: "No priority" },
+        { value: "1", sample: "1", label: "Urgent" },
+        { value: "2", sample: "2", label: "High" },
+        { value: "3", sample: "3", label: "Medium" },
+        { value: "4", sample: "4", label: "Low" },
+      ],
+    },
+    {
+      required: false,
+      label: "Label",
+      key: "labelId",
+      helpText: "Label which was assigned to the issue.",
+      dynamic: "label.id.name",
+      altersDynamicFields: true,
+    },
+    {
+      required: false,
+      label: "Project",
+      // This key is snake-cased for backwards compatibility with the dynamic field selector for project milestone
```

Comment:
Not sure I follow, backwards compatible how – this is brand new trigger?

Reply:
Look at the `projectMilestone` trigger under `src/triggers` that we use to populate the dynamic selector

Reply:
Could update that to support either/or I guess rather than compromising this new one 🤔

Reply:
That one is used in the polling trigger version and renaming a bundle variable is a breaking change. I could just create v2 instead to use here?

Reply:
Oh, I see what you mean: check for either variable there